llvm/test/CodeGen/AMDGPU/addrspacecast.ll

```diff
@@ -215,11 +215,14 @@ define amdgpu_kernel void @use_flat_to_constant_addrspacecast(ptr %ptr) #0 {
 }
 
 ; HSA-LABEL: {{^}}cast_0_group_to_flat_addrspacecast:
+; CI: s_load_dword [[APERTURE:s[0-9]+]], s[4:5], 0x10
```

Comment:
I just recover these lines from the commit before `AAAddressSpace` was enabled in `AMDGPUAttributorPass`.